app/models/comment.rb

```diff
@@ -1,2 +1,9 @@
 class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_save :update_comments_counter
+  def update_comments_counter
+    post.increment!(:comments_counter)
+  end
```

Comment:
- [ ] (Optional) It would be nice to make this function a _**private**_ one. 

```
private 

def update_comments_counter
  write code
end
```

#### When you make functions private, you prevent the function from being accessible by others and that way, you prevent malicious attacks. 👍